internal/consensus/reactor.go

```diff
@@ -61,10 +62,11 @@ func NewReactor(consensusState *State, waitSync bool, options ...ReactorOption)
 	conR := &Reactor{
 		conS:          consensusState,
 		waitSync:      atomic.Bool{},
-		rs:            consensusState.GetRoundState(),
-		initialHeight: consensusState.state.InitialHeight,
+		rs:            consensusState.RoundState,
```

Comment:
Not sure if I understood the change here.

Reply:
Reverted this change, I removed the getter at one point but brought it back and this diff still persisted